app/build.gradle

```diff
@@ -37,14 +37,14 @@ dependencies {
     implementation "android.arch.work:work-runtime:$work_version"
```

Comment:
New version for this library is available.

Reply:
But the implementation of it requires changes. Worker constructor requires 2 parameters or so. Hence, I reverted back. What do I do ?

Reply:
Done. :+1: